src/custom/components/TransactionSettings/TransactionSettingsMod.tsx

```diff
@@ -143,13 +146,27 @@ export default function TransactionSettings({ placeholderSlippage }: Transaction
     setDeadlineError(false)
 
     if (value.length === 0) {
+      ReactGA.event({
+        category: 'Order Expiration Time',
+        action: 'Set Default Expiration Time',
+        value: DEFAULT_DEADLINE_FROM_NOW,
+      })
       setDeadline(DEFAULT_DEADLINE_FROM_NOW)
     } else {
       try {
         const parsed: number = Math.floor(Number.parseFloat(value) * 60)
-        if (!Number.isInteger(parsed) || parsed < MINIMUM_ORDER_VALID_TO_TIME_SECONDS || parsed > 180 * 60) {
+        if (
+          !Number.isInteger(parsed) || // Check deadline is a number
+          parsed < MINIMUM_ORDER_VALID_TO_TIME_SECONDS || // Check deadline is not too small
+          parsed > MAX_DEADLINE_MINUTES * 60 // Check deadline is not too big
+        ) {
           setDeadlineError(DeadlineError.InvalidInput)
         } else {
```

Comment:
Do you guys think it's worth tracking invalid inputs here to paint a picture of what kinds of times outside our allowed range ppl want?

Reply:
Not sure, i wouldn't think is too important flow to capture